tests/cpp/test_multidevice_sharding.cpp

```diff
@@ -16,67 +16,68 @@
 namespace nvfuser {
 
 // params: concrete vs symbolic input, sharded axis
-class MultideviceShardingTest
+class MultiDevicePointwiseTest
```

Comment:
This test isn't a pointwise test though?

Reply:
Correct. Renamed to MultiDeviceReductionTest.